compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/linalg_function_outlining.mlir

```diff
@@ -178,32 +208,3 @@ func.func @linalg_fill_copy(%A: memref<4xf32>, %B: memref<4xf32>) {
   }
   return
 }
-
-// -----
-
-func.func @unsupported_linalg_op(%A: memref<4x8xbf16>, %B: memref<8x4xbf16>, %C: memref<4x4xf32>) {
-  %c2 = arith.constant 2 : index
-  %c1 = arith.constant 1 : index
-  %tile = amdaie.tile(%c1, %c2)
-  %1 = amdaie.core(%tile, in : [], out : []) {
-    // expected-error@+1 {{unsupported linalg op for outlining}}
```

Comment:
Couldn't find a way to selectively enable `expected-error` (besides just creating two separate files for the two `RUN`s). Need to ask around.